src/KOKKOS/fix_rx_kokkos.cpp

```diff
@@ -1389,9 +1389,9 @@ void FixRxKokkos<DeviceType>::operator()(Tag_FixRxKokkos_solveSystems<ZERO_RATES
     // Store the solution back in dvector.
     for (int ispecies = 0; ispecies < nspecies; ispecies++)
     {
-      if (y[ispecies] < -MY_EPSILON)
+      if (y[ispecies] < -1.0e-10)
```

Comment:
@stonecp , can you work with Jim to determine what this limit should really be here (and in other places in fix_rx*.cpp?  And then make a proper fix with a #define or const in a header/top-of-cpp-file location?  Thanks.